static_src/components/user_list.jsx

```diff
@@ -134,8 +134,6 @@ export default class UserList extends React.Component {
 
     if (this.props.empty) {
       content = this.emptyState;
-    } else if (this.props.users.length === 1) {
```

Comment:
leave it in for now. we can revisit this when we develop the UI for the space managers to move exiting org users into the space.

if it's a hassle to maintain that separate `onlyOneState` state, let's talk about it then though.